src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialVideosPageObject.java

```diff
@@ -114,33 +104,47 @@ public void deleteVideo() {
 
   public void verifyDeleteViaGlobalNotifications() {
     YoutubeVideo video = YoutubeVideoProvider.getLatestVideoForQuery("truth");
-
+    System.out.println("Video: " + video.getTitle());
+    System.out.println("Get URL: " + video.getUrl());
     addVideoViaAjax(video.getUrl());
-    deleteVideo();
+    deleteNewestVideo();
     String deletedVideo = "\"File:" + video.getTitle() + "\" has been deleted. (undelete)";
+    System.out.println("BannerNotificationText: " + getBannerNotificationText());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS106)